### PR TITLE
[CORE-1494] Support sidecar_resource_requests in pipeline spec

### PR DESCRIPTION
**Pull request by armaanv**

### CODE REVIEW

File: src/server/pps/pretty/pretty.go [hunk 1]
```diff
@@ -286,6 +286,9 @@ ParallelismSpec: {{.Details.ParallelismSpec}}
 {{ if .Details.SidecarResourceLimits }}SidecarResourceLimits:
   CPU: {{ .Details.SidecarResourceLimits.Cpu }}
   Memory: {{ .Details.SidecarResourceLimits.Memory }} {{end}}
+{{ if .Details.SidecarResourceRequests }}SidecarResourceRequests:
```
Comment (jrockway) [inline src/server/pps/pretty/pretty.go:289]: We should print requests before limits, to be consistent with the non-sidecar stuff above.

**Review by jrockway — COMMENTED** [review]
There should be a way to set the defaults from the helm chart.

**Comment by armaanv** [comment]
> There should be a way to set the defaults from the helm chart.

yup, meant to add this in the original PR. It's in the latest commit

**Review by jrockway — APPROVED** [review]
Very nice!

**Pull request merged**